src/Rsk.IdentityServer.Migration/Migrator.cs

```diff
@@ -31,11 +32,14 @@ public class Migrator
 
         public async Task Migrate()
         {
-            var clients = await clientReader.Read();
-            var scopes = await scopeReader.Read();
+            var clients = clientReader.Read();
+            var scopes = scopeReader.Read();
+           
+            await clientWriter.Write(clients.Select(x => x.ToDuende()).ToList());
+
+            var result = scopes.GetApiResourcesAndApiScopes();
```

Comment:
**suggestion:** complete all of your mapping before writing anything to the database